tests/phpunit/elementor/core/data/test-manager.php

```diff
@@ -271,7 +266,7 @@ public function test_commands_formats() {
 		$this->manager->run_server();
 
 		$this->assertEquals( [
-			'globals/index' => 'globals/index',
+			'globals/index' => 'globals/{id}',
```

Comment:
How it changed?

Reply:
Prev 'globals/index' was registered by wordpress register_route, which does not support our format system.
after moving to endpoints 'internal' which is now called 'index endpoint' it does support get_item and the format is required to reach get_item via `Data\Manager::run();`, the format is '{id}'.

Reply:
BC

Reply:
Handled: https://github.com/elementor/elementor/pull/12820/commits/57310f6150c9f03cd50ce1c4c27799960a1a2088